docs/5___ssd.adoc

```diff
@@ -678,7 +681,12 @@ When the URI is a same-document URI with a fragment identifier, for example #oth
 
 Note that implementations are only *REQUIRED* to support relative URIs as specified above, and that especially relative URIs that move beyond the baseURI (i.e. go "up" a level via ..) are *not required* to be supported by implementations, and are in fact often not supported for security or other reasons. Implementations are also *not required* to support any absolute URIs and any specific URI schemes (but are of course allowed to support any and all kinds of URIs where this is considered useful).
 
-{empty}[ Since the release of SSP 1.0, the need to support the exchange of system structure descriptions containing components with no specified implementation has been identified, to exchange system designs as templates, for example. Future releases of SSP will therefore likely make the source attribute optional, to support such use cases. Current practice for 1.x has been to either already treat this attribute as optional or to use the empty string value to indicate a missing implementation. Tools wanting to support these use cases should therefore be prepared to accept SSD files with missing or empty source attributes on components, and treat them like empty systems for the purposes of semantics. ]
+If the source attribute is missing, this indicates that there is no provided source for the component, indicating a simulation architecture design without implementation.
```

Comment:
Small clarification: "indicating a simulation architecture design without a complete and executable implementation"

Reply:
Adjusted to `complete executable specification`